mep/people/templates/people/member_list.html

```diff
@@ -20,6 +20,23 @@
         <label class="query">
             {% render_field form.query aria-label=form.query.label %}
         </label>
+        <fieldset>
+                <legend>Filter by:</legend>
+                <div class="toggle-filters">
+                    {% render_field form.has_card aria-label=form.has_card.label class+="sr-only sr-only-focusable" %}
+                    <label for="id_has_card">
+                        {# Wrapping attribute in <span> for CSS targeting #}
```

Comment:
this is definitely the right move and i feel like it's common enough that it doesn't even necessarily warrant a comment (though I appreciate it). will definitely be popping up lots of other places throughout this codebase.